internal/server/sshd.go

```diff
@@ -144,21 +145,19 @@ func StartSSHServer(sshListener net.Listener, privateKey ssh.Signer, insecure bo
 	log.Printf("Loading authorized keys from: %s\n", authorizedKeysPath)
 	authorizedControllers, err := readPubKeys(authorizedKeysPath)
 	if err != nil {
-		log.Fatal(err)
+		log.Println("read authorizekeys fail :", err)
```

Comment:
This will break things that expect authorizedControllers to be non-nil/non-empty